extensions/ql-vscode/src/query-history.ts

```diff
@@ -286,9 +286,10 @@ export class QueryHistoryManager {
   }
 
   async handleOpenQuery(
-    singleItem: CompletedQuery,
-    multiSelect: CompletedQuery[]
+    s: CompletedQuery,
```

Comment:
Why make these variable names less legible?

Reply:
Fine the way it is, I suppose.

Reply:
Fixed in the latest version.